plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -456,9 +363,9 @@ def showControlInterface(self):
     ##  Send a command to printer. 
     #   \param cmd string with g-code
     def sendCommand(self, cmd):
-        if self.isPrinting():
+        if self._progress:
```

Comment:
I could change this, but it would only be reimplementing data that's implied by the progress. (-1 && 0 == not printing, anything else is doing something).